Tools/GenerateWebsiteData.ps1

```diff
@@ -50,6 +50,17 @@ function getParameterSets ($docObj) {
     return $sets
 }
 
+# getLinks creates the related link hashtable for each cmdlet.
+# It takes in the get-help object for the cmdlet.
```

Comment:
Is "Get-Help object" accurate? What is the actual type name? e.g. `System.Management.Automation.CmdletHelp`?